module/behaviour/planning/SimpleWalkPathPlanner/src/SimpleWalkPathPlanner.cpp

```diff
@@ -245,8 +242,9 @@ namespace module::behaviour::planning {
 
                 float scaleS         = 2.0 / (1.0 + std::exp(-a * std::fabs(position.y()) + b)) - 1.0;
                 float scaleS2        = angle / M_PI;
-                float finalSideSpeed = -speedFactor * ((0.0 < position.y()) - (position.y() < 0.0)) * sideStep
-                                       * sideSpeed * scaleS * (1.0 - scaleS2);
+                float finalSideSpeed = -speedFactor
+                                       * (static_cast<int>(0.0 < position.y()) - static_cast<int>(position.y() < 0.0))
```

Comment:
Here is `position.y()` an integer?

Reply:
This change didn't make it through the merge. If clang-tidy suggests it again after this PR, we can review that next PR.